### DISCUSSION TITLE
Tasmota tuya covert

### DISCUSSION THREAD

**Discussion by fabian-tasmota** [ct-Open-Source/tuya-convert#1036] [Ideas]
could you release a new version with tasmota version 11.0.0?

**Comment by MacSass** [comment]
You should be able to just replace the binary tasmota file in the "files" folder with a newer version (tasmota-minimal.bin).
Or you just upgrade after the flash (first upgrade to full version, then to 11.0.0 minimal then to 11.0.0 full version)

**Comment by fabian-tasmota** [comment]
how do I add the new version? does German bin also work?

**Reply by MacSass** [reply]
Just replace the .bin file with the "minimal" version you desire ... if you are uncomfortable with that, I would recommend to use what is there right now and upgrade independently after flashing ... you can alwas do that later to whatever version you desire.

**Comment by fabian-tasmota** [comment]
I flash several devices that would take far too long

**Comment by fabian-tasmota** [comment]
Do I have to copy that and then replace the bin and upload it myself?
